data/mundo/cpsAssets/23263889.json

```diff
@@ -398,7 +398,29 @@
         "tile": "IDT2 Include",
         "href": "/idt2/cb1a5166-cfbb-4520-bdac-6159299acff6",
         "platform": "highweb",
-        "type": "include"
+        "type": "include",
```

Comment:
The example include looks very skewed, it also seems it's been converted into an image. Is that intentional?

<img width="877" alt="Screenshot 2020-06-02 at 09 01 36" src="https://user-images.githubusercontent.com/4706195/83495648-e23b7180-a4af-11ea-9236-4a8275ad70f4.png">

Reply:
IDT2s on AMP should be converted to images but should not look skewed I would say. I wonder if the correct image is being used for the breakpoint you viewed at @HarveyPeachey ?

Reply:
This was viewed on my desktop so whichever group that is. The image was also skewed when I reduced the window size to a single column layout.

Reply:
@HarveyPeachey @andrewscfc I also noticed they looked skewed when adjusting between breakpoints however if you adjust to a certain breakpoint and refresh the browser, that goes away for that. breakpoint. Wasn't entirely sure what the cause of this was

Reply:
@rhenshaw56 So the skewed image happens on first page load for me. I'm viewing it incognito in chrome. It's also skewed on every gel group breakpoint

Reply:
I noticed that setting the `amp-img`'s `min-width` instead of `width` as it is currently being done fixes the problem by giving the image room to grow, preventing it being skewed

Reply:
This might be something we want addressed in a separate issue.

Reply:
@andrewscfc @HarveyPeachey @mykeels this should be fixed now. can you please confirm that?